src/test/common/steps/check-details.js

```diff
@@ -68,6 +68,11 @@ When(/^I choose to change my answer to Do you live in Scotland$/, async function
   await pages.check.clickChangeLinkFor('Do you live in Scotland?')
 })
 
+When(/^I click on the back link$/, async function () {
```

Comment:
This is inconsistent with how we've done it on other stories, we have a function in `common-assertions.js` called `assertBackLinkPointsToPage` which we should probably use instead of this. I also think that a step like this should probably be in `common-steps.js` too, and should be calling a function in `Page.js` rather than something specific to the Check Details page.

Reply:
Updated to now just assert that the back link url is correct using the `assertBackLinkPointsToPage` method.